backend/src/main/java/com/redhat/cloud/notifications/db/repositories/EventTypeRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.cloud.notifications.db.repositories;
+
+import com.redhat.cloud.notifications.models.EventType;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.ws.rs.NotFoundException;
+
+@ApplicationScoped
+public class EventTypeRepository {
+
+    @Inject
+    EntityManager entityManager;
+
+    /**
+     * Find an event type by its exact name.
+     * @param eventTypeName the name of the event type to look by.
+     * @return the found event type.
+     */
+    public EventType findByName(final String eventTypeName) {
```

Comment:
The `name` of an event type is not a unique identifier if it is not associated with an app `name` or `id`.

`app_name_1 / event_type_name_1` can coexist with `app_name_2 / event_type_name_1` in the DB.

Reply:
@g-duval created a similar method in https://github.com/RedHatInsights/notifications-backend/pull/1637. Both methods may or may not do the same thing depending on the final signature of this one.

Reply:
This is no longer relevant, since the method is not required.